custom/icds_reports/utils/aggregation_helpers/distributed/migration_forms.py

```diff
@@ -0,0 +1,81 @@
+from dateutil.relativedelta import relativedelta
+
+from custom.icds_reports.const import AGG_MIGRATION_TABLE
+from custom.icds_reports.utils.aggregation_helpers import month_formatter
+from custom.icds_reports.utils.aggregation_helpers.distributed.base import (
+    StateBasedAggregationDistributedHelper,
+)
+
+
+class MigrationFormsAggregationDistributedHelper(StateBasedAggregationDistributedHelper):
```

Comment:
you will also need to add this migration to the Airlfow Pipes [repo](https://github.com/dimagi/pipes).
You can do that by creating an aggregation function of this in [this file like other agg fuctions](https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/tasks.py#L667) and first call that function in `move_ucr_data_into_aggregation_tables` function in similar way others are being called so that its being called during local test cases and then also add that function to [here](https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/management/commands/run_aggregation_query.py) and also update the Pipes report. Example PR could be [this](https://github.com/dimagi/pipes/pull/24/files)